### DISCUSSION TITLE
Third party Text to Speech voices populate on the list but don't work

### DISCUSSION THREAD

**Discussion by AdeonWriter** [vrcx-team/VRCX#353] [General]
VRCX loads in all of Window's installed voices, but the third party ones don't seem to work.

They work in every application besides VRCX; not sure if it's because other apps like Discord don't let you pick the voice - they just respect whichever one you set in windows TTS settings.

![image](https://user-images.githubusercontent.com/2362544/171965167-4e7f6cc3-107a-4187-800a-bcfddf7b0a2f.png)

**Comment by Natsumi-sama** [comment]
https://github.com/pypy-vrc/VRCX/issues/239#issuecomment-877860061
VRCX should display the same voices as chrome.
https://github.com/pypy-vrc/VRCX/issues/325#issuecomment-1079553900

**Reply by AdeonWriter** [reply]
It does, the third party ones just don't have audio.

**Reply by Natsumi-sama** [reply]
Give my script a try linked in the comments, chrome/vrcx will need to be restarted for it to take effect.

**Reply by AdeonWriter** [reply]
Yeah! Mine was already on the list to begin with, sadly running your script did not make actually selecting it work - no voice.

**Reply by Natsumi-sama** [reply]
How annoying, I wish chrome didn't force the new Windows TTS API that no one uses.

**Reply by AdeonWriter** [reply]
Oddly enough, it does work in chrome/discord
